content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md

```diff
@@ -4,17 +4,18 @@ reviewers:
 title: Upgrading kubeadm clusters
 content_template: templates/task
 weight: 20
-min-kubernetes-server-version: 1.18
+min-kubernetes-server-version: 1.19
 ---
 
 {{% capture overview %}}
 
 This page explains how to upgrade a Kubernetes cluster created with kubeadm from version
-1.17.x to version 1.18.x, and from version 1.18.x to 1.18.y (where `y > x`).
+1.18.x to version 1.19.x, and from version 1.19.x to 1.19.y (where `y > x`).
 
 To see information about upgrading clusters created using older versions of kubeadm,
 please refer to following pages instead:
 
+- [Upgrading kubeadm cluster from 1.17 to 1.18](https://v1-18.docs.kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/)
 - [Upgrading kubeadm cluster from 1.16 to 1.17](https://v1-17.docs.kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/)
 - [Upgrading kubeadm cluster from 1.15 to 1.16](https://v1-16.docs.kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/)
 - [Upgrading kubeadm cluster from 1.14 to 1.15](https://v1-15.docs.kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-15/)
```

Comment:
@neolit123 should we drop the link "from 1.14 to 1.15"?

Reply:
If v1.14 is still supported after the v1.19 release happens, I would expect to leave that link in.